oeq-ts-rest-api/src/Utils.ts

```diff
@@ -25,23 +25,26 @@ import { cloneDeep } from 'lodash';
  * @param recursive True if processing nested objects is required.
  * @param converter A function converting fields' type.
  */
-const convertFields = <T, R>(
-  input: unknown,
+const convertFields = <R>(
+  input: Record<string, unknown>,
   targetFields: string[],
   recursive: boolean,
-  converter: (value: T) => R
+  converter: (value: unknown) => R
 ): void => {
-  const entries: [string, any][] = Object.entries(input as any);
+  const entries: [string, unknown][] = Object.entries(input);
```

Comment:
The typing here may be optional, testing on the ts playground it looks like `[string, unknown][]` is inferred https://www.typescriptlang.org/play?#code/MYewdgzgLgBApgDwIYFsAOAbOAuGAlOUAJwBMAeaIgSzAHMAaGAVzAGswQB3MAPhgF4YAbwC+AKDGhIsKHGgCYAeQBGAK0JQAdHDBRqcgBSJUmOAEogA

```suggestion
  const entries = Object.entries(input);
```

Reply:
Ah nice. I'll take it for a spin. Thanks.

Reply:
Yes, with that done it did kind reduce readability further down, so I simply merged the lines. All done. :+1: